deploy/overlays/prod/patch_deployment.yaml

```diff
@@ -31,8 +31,13 @@ spec:
         - name: redis-config
           configMap:
             name: redis-config
+        - name: ai-config
```

Comment:
Maybe should move this to $ROOT/deploy/base/deployment.yaml? So this will be a shared pattern. Then a lot of patch above can be saved.